integration_test/config/itest.yml

```diff
@@ -30,7 +30,8 @@ dataset_options:
   public_router_ipv4_address: 172.16.90.4
   physical_network_simulated_ip: 10.210.0.2
   physical_network_vm1_ip: 10.210.0.104
-  physical_network_vm7_ip: 10.210.0.105
+  physical_network_vm5_ip: 10.210.0.105
```

Comment:
It's for testing promiscuous and wanedge using GRE tunnels, while we used to only test with MAC2MAC.

Those tests involving vm5 are however marked as pending as there is an issue with GRE tunnels, and fixing that is outside of the scope of this pull request.